apps/antalmanac/src/lib/termData.ts

```diff
@@ -81,4 +90,19 @@ function getDefaultTerm() {
     return termData[defaultTerm];
 }
 
-export { defaultTerm, getDefaultTerm, termData };
+function getDefaultFinalsStart() {
+    return termData[defaultTerm].finalsStartDate;
+}
+
+/**
+ * Defaults to finals for Winter 2024
+ * Days offset by 1 to accomodate toggling with Saturday finals
+ */
+function getDefaultFinalsStartDate() {
+    const defaultFinals = termData[defaultTerm].finalsStartDate;
+    return defaultFinals
+        ? new Date(defaultFinals[0], defaultFinals[1], defaultFinals[2] + 1)
+        : new Date(2024, 2, 16 + 1);
+}
```

Comment:
```suggestion
function getDefaultFinalsStartDate(): Date | undefined {
    const [year, month, day] = termData[defaultTerm].finalsStartDate || [];

    return year && month && day ? new Date(year, month, day + 1) : undefined;
}
```
Additionally, it might be better to refactor so that you're not accessing a "magic" array -- all values should (ideally) tell you what they are